schema/legacy/resolvers/Location.ts

```diff
@@ -41,6 +41,17 @@ export const Location: LocationResolvers = {
     const children = await ctx.orm.location.loadMany(childIds.map(c => c.id));
     return children.filter(isValue);
   },
+  async geofence(parent, _, ctx) {
+    const hack = await ctx.orm.location.load(decodeGlobalId(parent.id).id);
+
+    if (!hack.latitude || !hack.longitude || !hack.radius) return undefined;
```

Comment:
probably need to explicitly check for `radius === null` or whatever, since if `radius === 0`, `!radius` will be true

Reply:
+1

You can use nullish from util.ts